app/code/Magento/Wishlist/Block/Share/Email/Items.php

```diff
@@ -11,17 +11,57 @@
  */
 namespace Magento\Wishlist\Block\Share\Email;
 
+use Magento\Catalog\Model\Product;
+use Magento\Catalog\Model\Product\Configuration\Item\ItemResolverInterface;
+use Magento\Catalog\Model\Product\Image\UrlBuilder;
+use Magento\Framework\View\ConfigInterface;
+use Magento\Wishlist\Model\Item;
+
 /**
  * @api
  * @since 100.0.2
  */
 class Items extends \Magento\Wishlist\Block\AbstractBlock
 {
+    /** @var ItemResolverInterface */
```

Comment:
Please move the docblock to a new line.
```suggestion
    /**
     * @var ItemResolverInterface
     */
```